src/server/database/Database/Implementation/HotfixDatabase.cpp

```diff
@@ -175,6 +175,10 @@ void HotfixDatabaseConnection::DoPrepareStatements()
     PrepareStatement(HOTFIX_SEL_CINEMATIC_SEQUENCES, "SELECT ID, SoundID, Camera1, Camera2, Camera3, Camera4, Camera5, Camera6, Camera7, Camera8"
         " FROM cinematic_sequences ORDER BY ID DESC", CONNECTION_SYNCH);
 
+    // ConversationLine.db2
+    PrepareStatement(HOTFIX_SEL_CONVERSATION_LINE, "SELECT ID, Unk1, Unk2, Unk3, NextLineID, Unk4, UnkBoolean, Unk5, Unk6"
+        " FROM conversation_line ORDER BY ID DESC", CONNECTION_SYNCH);
```

Comment:
Sorry, I was wrong. The correct table name here is `conversation_template`.
Fix rework still in progress.